plugins/tangle/solidifier_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/iotaledger/goshimmer/packages/database"
+	goshimmerDB "github.com/iotaledger/goshimmer/packages/database"
```

Comment:
Remove unnecessary import alias

Reply:
Removed

Reply:
But you didn't push :stuck_out_tongue_winking_eye:

Reply:
I'll fix this so that we can merge